main/SCsub

```diff
@@ -25,7 +26,5 @@ env.CommandNoCache("#main/splash_editor.gen.h", "#main/splash_editor.png", run_i
 env.Depends("#main/app_icon.gen.h", "#main/app_icon.png")
 env.CommandNoCache("#main/app_icon.gen.h", "#main/app_icon.png", run_in_subprocess(main_builders.make_app_icon))
 
-SConscript('tests/SCsub')
```

Comment:
With all due respect, I'm not sure why we're not using this SCons script. Was there a error of some sort that was only fixed when you did this?